specs/mediawiki/v1/content.yaml

```diff
@@ -445,6 +445,69 @@ paths:
       x-backend-request:
         uri: /{domain}/sys/page_revisions/rev/{revision}
 
+  /{module:page}/wikitext/{title}{/revision}:
+    post:
+      tags:
+        - Page content
+      description: >
+        Save a new revision of a page
+
+        For new pages, the `revision` parameter should be left empty.
+        For editing existing pages, it should contain the revision ID
+        the edit is derived from.
+
+        Stability: [unstable](https://www.mediawiki.org/wiki/API_versioning#Unstable)
+      parameters:
+        - name: title
+          in: path
+          description: The page title.
+          type: string
+          required: true
+        - name: revision
```

Comment:
I put it in the URI as the logic (for me) is - *the POST request is based on this revision, which in turn creates a new one*. Had we have PUT requests, then, yes, the revision shouldn't be in the path. It all really depends on the way we look at it, I guess.